examples/websocket-retry/websocket_service.bal

```diff
@@ -0,0 +1,27 @@
+import ballerina/http;
+import ballerina/log;
+
+// Annotations specifying the webSocket service.
```

Comment:
This annotation does not specify a WebSocket service, does it? IMO, we should improve this comment.